docs/contributing.md

```diff
@@ -36,3 +36,17 @@ $ make verify
 
 $ make install.tools
 - Install tools used by verify"
+
+## Debug Hints
+To send a stackdump request via signal to the capship server:
+```
+$ sudo kill -SIGUSR1 `pidof capship`
```

Comment:
Is `pidof` a real command or an indication that the user replaces this section with a pid?

also, `$()` over backticks, but that's my nit.

Reply:
it's a real command... it works well... no filling in required...
`$() over backtics?`